plugins/UsersManager/templates/index.twig

```diff
@@ -191,7 +209,13 @@
                         <td id="password" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>-</td>
                         <td id="email" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.email }}</td>
                         <td id="alias" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.alias|raw }}</td>
-                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">{{ user.token_auth|slice(0, 8) }}…</td>
+                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">
```

Comment:
after you mention it, absolutely agree :+1:  

@mneudert to confirm,  we'd like to do the following:

* Remove the token_auth listed in the UI (when logged in as Super User) -> improves security as the page may be transferred over non secure networks, or may be cached in computers, etc. 
* Remove possibility for super users to regenerate tokens of other users.